core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -179,6 +182,14 @@ angular.module('oppia').component('preferencesPage', {
           // No further action is needed.
         });
       };
+
+      ctrl.addFocusWithoutScroll = function(label) {
+        FocusManagerService.setFocus(label);
+        setTimeout(function() {
+          window.scrollTo(0, 0);
+        }, 5);
```

Comment:
Ditto: auto scroll

Reply:
we dont want it to scroll in small screen